### PR TITLE
[AIR] Remove unnecessary pandas from examples

### PR DESCRIPTION
**Pull request by Yard1**
## Why are these changes needed?

Removes unnecessary pandas usage from AIR examples. Helps ensure users do not follow bad practices.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/ray-air/doc_code/air_key_concepts.py [hunk 1]
```diff
@@ -75,10 +75,8 @@
 batch_predictor = BatchPredictor.from_checkpoint(result.checkpoint, XGBoostPredictor)
 
 # Bulk batch prediction.
-predicted_labels = (
-    batch_predictor.predict(test_dataset)
-    .map_batches(lambda df: (df > 0.5).astype(int), batch_format="pandas")
-    .to_pandas(limit=float("inf"))
+predicted_labels = batch_predictor.predict(test_dataset).map_batches(
```
Comment (ericl) [inline doc/source/ray-air/doc_code/air_key_concepts.py:78]: How about we drop the map_batches and then call `predicted_labels.show()` on the next line? We shouldn't complicate examples.

File: doc/source/ray-air/doc_code/tf_starter.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ def train_func(config: dict):
 
 predictions = batch_predictor.predict(prediction_dataset, dtype=tf.float32)
 
-pandas_predictions = predictions.to_pandas(float("inf"))
+print(f"PREDICTIONS")
```
Comment (ericl) [inline doc/source/ray-air/doc_code/tf_starter.py:108]:
```suggestion
print("Predictions:")
```

File: doc/source/ray-air/doc_code/xgboost_starter.py [hunk 3]
```diff
@@ -59,17 +59,13 @@
 
 batch_predictor = BatchPredictor.from_checkpoint(result.checkpoint, XGBoostPredictor)
 
-predicted_labels = (
-    batch_predictor.predict(test_dataset)
-    .map_batches(lambda df: (df > 0.5).astype(int), batch_format="pandas")
-    .to_pandas(limit=float("inf"))
+predicted_labels = batch_predictor.predict(test_dataset).map_batches(
+    lambda df: (df > 0.5).astype(int), batch_format="pandas"
```
Comment (ericl) [inline doc/source/ray-air/doc_code/xgboost_starter.py:63]: Drop the filter as above?

**Review by ericl — APPROVED** [review]

File: doc/source/ray-air/examples/upload_to_wandb.ipynb [hunk 4]
```diff
@@ -69,11 +68,8 @@
    "outputs": [],
    "source": [
     "def get_train_dataset() -> ray.data.Dataset:\n",
-    "    \"\"\"Return the \"Breast cancer\" dataset as a Ray dataset.\"\"\"\n",
-    "    data_raw = load_breast_cancer(as_frame=True)\n",
-    "    df = data_raw[\"data\"]\n",
-    "    df[\"target\"] = data_raw[\"target\"]\n",
-    "    return ray.data.from_pandas(df)"
+    "    dataset = ray.data.read_csv(\"s3://air-example-data/breast_cancer.csv\")\n",
```
Comment (richardliaw) [inline doc/source/ray-air/examples/upload_to_wandb.ipynb:71]: so -- does this work if you do NOT have any aws credentials configured?

Comment (richardliaw) [reply doc/source/ray-air/examples/upload_to_wandb.ipynb:71]: (please verify!)

Comment (Yard1) [reply doc/source/ray-air/examples/upload_to_wandb.ipynb:71]: It does, I tried it on Colab:
Screenshot 2022-06-24 at 08 20 38

**Pull request merged**